File: CStoreManagement/app/src/main/java/com/c_store/zhiyazhang/cstoremanagement/TestActivity.java
package com.c_store.zhiyazhang.cstoremanagement;

import android.os.Bundle;
import android.support.annotation.Nullable;
import android.view.View;
import android.widget.Button;
import android.widget.Toast;

import com.c_store.zhiyazhang.cstoremanagement.utils.activity.MyActivity;

import butterknife.BindView;

/**
 * Created by zhiya.zhang
 * on 2017/6/2 10:15.
 */

public class TestActivity extends MyActivity {


    @BindView(R.id.button)
    Button button;

    @Override
    protected void onCreate(@Nullable Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        button.setOnClickListener(new View.OnClickListener() {
            @Override
            public void onClick(View v) {
                Toast.makeText(TestActivity.this, "test", Toast.LENGTH_SHORT).show();
            }
        });
    }

    @Override
    protected int getLayoutId() {
        return R.layout.test;
    }
}

File: KotlinCStoreManagement/app/src/main/java/com/cstore/zhiyazhang/cstoremanagement/TestActivity.kt
package com.cstore.zhiyazhang.cstoremanagement

import android.os.Bundle
import android.widget.Toast

import com.cstore.zhiyazhang.cstoremanagement.utils.MyActivity

/**
 * Created by zhiya.zhang
 * on 2017/6/7 14:40.
 */

class TestActivity(override val layoutId: Int = R.layout.app_bar_home) : MyActivity() {
    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        val x="asdflksajer"
        if("b" in x){
            Toast.makeText(this@TestActivity, "有", Toast.LENGTH_SHORT).show()
        }else
            Toast.makeText(this@TestActivity, "没有", Toast.LENGTH_SHORT).show()
    }
}
